src/test/java/com/battleship/backend/validators/PlaceShipsValidatorTest.java

```diff
@@ -1,9 +1,11 @@
 package com.battleship.backend.validators;
 
 import com.battleship.backend.TestClasses;
+import com.battleship.backend.models.Board;
 import com.battleship.backend.models.Boardable;
 import com.battleship.backend.models.Request;
 import com.battleship.backend.models.Ship;
+import org.apache.logging.log4j.message.ReusableMessage;
```

Comment:
👀  https://www.cnn.com/2021/12/15/tech/log4j-vulnerability/index.html

I don't think using log4j in your project has any repercussions though.

Reply:
Removed it anyway, wasn't using it. Also removed other unnecessary import statements.